zowe/zlux-server-framework
# Add zcrypto library use for jceccaracfks support

To address https://github.com/zowe/community/issues/1851 for the app-server, the https://www.npmjs.com/package/zcrypto library uses gsk/systemssl for getting keyrings in node.
ZSS is also a user of gsk, and was tested as working for ICSF, so I am hopeful that switching away from our own keyring_js library towards this library will yield better long-term results for keyring support within the app-server.

In this PR, both libraries are still available, though only switchable via a const that is not yet exposed as a configuration item.

How to test:
In zowe's config, you want to use keyrings of either JCERACFKS type or of JCECCARACFKS.
We already support JCERACFKS, so we should see no regression (server loads and is accessible for a url such as the desktop or /plugins)
If we're successful with JCECARACFKS, we should see the same behavior, whereas before we could not load the key/cert at all.
